src/NuGet.Core/NuGet.Protocol/Plugins/PluginProcess.cs

```diff
@@ -136,7 +136,7 @@ public void Kill()
             UpdateIdIfNecessary();
         }
 
-        internal void Start()
+        public void Start()
```

Comment:
Actually, what's preventing you to use the Plugin Factory?

Reply:
We are using our own (copy of) Plugin factory with a fix for https://github.com/NuGet/Home/issues/7511 - but that means a call to Start should be possible from the outside.

Reply:
I think we can make that change, but I guess there's still a case for having the start method be public.

thoughts @dtivel

Reply:
Make it public.